jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/Jar.java

```diff
@@ -23,13 +23,17 @@
 import com.google.cloud.tools.jib.cli.jar.JarFiles;
 import com.google.cloud.tools.jib.filesystem.TempDirectoryProvider;
 import com.google.cloud.tools.jib.plugins.common.logging.ConsoleLogger;
+import com.google.cloud.tools.jib.plugins.common.logging.SingleThreadedExecutor;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.concurrent.Callable;
 import picocli.CommandLine;
+import picocli.CommandLine.Model.CommandSpec;
 
 @CommandLine.Command(name = "jar", showAtFileInUsageHelp = true, description = "Containerize a jar")
 public class Jar implements Callable<Integer> {
+  @CommandLine.Spec CommandSpec spec = CommandSpec.create();
```

Comment:
Probably

```suggestion
  @CommandLine.Spec protected CommandSpec spec = CommandSpec.create();
```

I guess it can't be `private`?

Reply:
great catch. On second thought, I think it should be private since it's only accessed in this class.

Reply:
Appu put `globalOptions` as `protected`, so probably there's a reason for that. But I'm not sure about `CommandSpec`. Just do some actual `jib jar` run to confirm you see correct output.

Reply:
Ah good point, running ` jib --target=docker://jib-cli-image jar unknown.jar` returned the expected output : `[ERROR] The file path provided does not exist: unknown.jar`. I think it could be okay?